### PR TITLE
Feat/use sdk

### PR DESCRIPTION
**Pull request by Blu-J**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ verify: c-lightning.s9pk
 install: c-lightning.s9pk
 	embassy-cli package install c-lightning
 
-c-lightning.s9pk: manifest.yaml image.tar instructions.md $(ASSET_PATHS) scripts/embassy.js
+c-lightning.s9pk: manifest.yaml image.tar instructions.md $(ASSET_PATHS)
```
Comment (chrisguida) [inline Makefile:19]: why was this removed?

Comment (Blu-J) [reply Makefile:19]: Wasn't meant, just a sec

**Review by chrisguida — APPROVED** [review]
looks good
